docs/routing.rst

```diff
@@ -120,12 +120,27 @@ Maps, Rules and Adapters
       class, the `converters` parameter to the constructor has to be used
       instead.
 
+   .. attribute:: _matcher
```

Comment:
Let's leave this as an internal, non-public API. I don't want to say we're going to support both matchers going forward, as I don't know what future improvements will require.